### ISSUE TITLE
Capacitor 5 support

### ISSUE THREAD

**Issue by ihmpavel** [epicshaggy/capacitor-native-biometric#93]
[Capacitor 5](https://ionic.io/blog/announcing-capacitor-5) has been announced and released.

Unfortunately plugins made for Capacitor 4 [are not backwards compatible](https://github.com/ionic-team/capacitor/issues/6504) without few changes. That means users are enforced to stay on Capacitor 4.

The change looks pretty straightforward - https://capacitorjs.com/docs/updating/plugins/5-0

Community plugins with open PRs
 - https://github.com/capacitor-community/file-opener/pull/35/files
 - https://github.com/capacitor-community/barcode-scanner/pull/234/files

**Comment by wizevlad** [comment]
👀

**Comment by riderx** [comment]
I forked the plugin, made upgrade to v5 and set CI/CD to test and keep it update automatic.
https://github.com/Cap-go/capacitor-native-biometric
My Org Capgo will finance the maintenance of it.

**Comment by craPkit** [comment]
This seems to have been fixed with Capacitor CLI 5.0.2:  
https://github.com/ionic-team/capacitor/issues/6504#issuecomment-1541614266

**Comment by cardacci** [comment]
Do we have any update on the migration to Capacitor 5?

**Comment by kientrant1** [comment]
I can't find out any newest version that can support capacitor 5. Do we have any news on this thread?

**Comment by riderx** [comment]
You can use the Capgo version.
It’s still maintained and had some fix for android and iOS latest version.
I did a release this week for capacitor 6

**Comment by kientrant1** [comment]
Just have one concern. In case if switching to the Capgo version. How can we migrate existing biometric that is setup in by original version?

**Comment by kientrant1** [comment]
> You can use the Capgo version. It’s still maintained and had some fix for android and iOS latest version. I did a release this week for capacitor 6

Just have one concern. In case if switching to the Capgo version. How can we migrate existing biometric that is setup in by original version?

**Comment by riderx** [comment]
Have a try in local. For me biometric data are link to the app not the plugin so it should be builtin replace

**Comment by kientrant1** [comment]
> Have a try in local. For me biometric data are link to the app not the plugin so it should be builtin replace

Thanks @riderx. Just retried and I can see that Capgo version still can load biometric from Epicshaggy version. Maybe, I got the issue of un-sync data because using "Live Update" feature (only update Web part) in Ionic. Switching to Capgo version requires to update in Native souce as well.

**Comment by riderx** [comment]
Yes it cannot be live update, the native plugin had to have a different name so the same JS cannot contact it
